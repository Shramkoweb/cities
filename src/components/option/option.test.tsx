import * as React from "react";
import * as renderer from "react-test-renderer";

import Option from "./option";

it(`Goods component render correct`, () => {

  const component = renderer
    .create(<Option />)
    .toJSON();

  expect(component).toMatchSnapshot();
});
