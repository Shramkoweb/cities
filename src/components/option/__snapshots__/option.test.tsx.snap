// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Goods component render correct 1`] = `
<li
  className=" places__option"
  onClick={[Function]}
  tabIndex={0}
/>
`;
